frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewMetaData/TagRow.tsx

```diff
@@ -99,80 +96,77 @@ export const TagRow = ({ feature }: IFeatureOverviewSidePanelTagsProps) => {
         }
     };
 
+    const addTagButton = (
```

Comment:
Should this be a function component and rendered as `<AddTagButton/>` instead?

I thought of the ["Never call function component functions directly" react docs](https://react.dev/reference/rules/react-calls-components-and-hooks#never-call-component-functions-directly), but this is a variable, so I'm not sure whether it applies here or not 🤔

Reply:
This doesn't apply for things rendered into a variable. But thanks for pointing it out. I refactored it to a separate component